grails-app/taglib/org/pih/warehouse/LocalizationTagLib.groovy

```diff
@@ -0,0 +1,43 @@
+package org.pih.warehouse
+
+import org.grails.plugins.web.taglib.ValidationTagLib
+
+import org.pih.warehouse.core.localization.MessageLocalizer
+
+class LocalizationTagLib {
```

Comment:
This isn't currently used anywhere. I figured we'd want to experiment with the new messageLocalizer before applying it everywhere. Once we're comfortable that this flow works, we can switch the namespace of this class to 'g' and then everywhere that we have <g:message> will go through this taglib.